bluesky_queueserver/manager/start_manager.py

```diff
@@ -174,16 +176,23 @@ def start_manager():
                         default="127.0.0.1:9092")
     parser.add_argument("--profile_collection", "-p", dest="profile_collection_path",
                         type=str, help="Path to directory that contains profile collection.", )
+    parser.add_argument("--allowed_plans_and_devices", dest="allowed_plans_and_devices_path",
+                        type=str,
+                        help="Path to file that contains the list of allowed plans. "
```

Comment:
Should not it be wrapped into `(` on this line and `)` three lines below?

Reply:
I guess it is not necessary, otherwise it wouldn't work. I think there is no need for extra parenthesis here: parentheses enclosing the argument list are sufficient.

Reply:
Indeed. I was just extra-paranoid about the multi-line definition of a single-line string.